package.json

```diff
@@ -1,34 +1,37 @@
 {
   "name": "srcery-colors-website",
-  "version": "3.0.0",
+  "version": "4.0.0",
   "private": true,
-  "description": "Website for promoting the Srcery color scheme.",
-  "homepage": "https://srcery-colors.github.io",
+  "description": "Srcery theme landing page",
+  "homepage": "https://srcery.sh",
+  "scripts": {
+    "fetch": "node scripts/fetch_gh_data.mjs",
+    "start": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "clean": "rm src/github.json"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/srcery-colors/srcery-colors.github.io"
   },
   "license": "MIT",
   "author": "Srcery Team",
-  "scripts": {
-    "build": "NODE_ENV=production snowpack build",
-    "clean": "rm -fr ./dist",
-    "serve": "NODE_ENV=development snowpack dev"
-  },
-  "browserslist": [
-    "> 5%",
-    "last 2 version",
-    "not dead"
-  ],
-  "dependencies": {
-    "glightbox": "^3.2.0"
-  },
   "devDependencies": {
-    "@snowpack/plugin-postcss": "^1.4.3",
     "autoprefixer": "^10.4.14",
-    "postcss": "^8.4.27",
-    "postcss-import": "^15.1.0",
-    "snowpack": "^3.8.8",
-    "tailwindcss": "^3.3.3"
+    "eslint": "^8.42.0",
+    "htmlhint": "^1.1.4",
+    "lodash": "^4.17.21",
+    "octokit": "^2.1.0",
+    "postcss": "^8.4.24",
+    "vite": "^4.3.4"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.4.0",
+    "@srcery-colors/srcery-palette": "^1.0.4",
+    "alpinejs": "^3.12.2",
+    "devicon": "^2.15.1",
+    "highlight.js": "^11.8.0",
+    "tailwindcss": "^3.3.2"
```

Comment:
Tailwind is a devDep.

Reply:
I never can quite figure out what does where when its a website, not a package. I feel that sort of partitioning only makes sense in that context, but hey, rights right.

Reply:
Yeah, Ikr. 🙃  Just that I've been more considerable about these things of late.

Me understanding is that deps is more for what you need to run the code, depDeps are for what you e.g. use to build/lint etc., and peerDeps are stuff that other packages need if they include yours.  Ref. [docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#dependencies).

In regards to Tailwind, it's only used when compiling the CSS before bundling.

Reply:
Makes sense, that's more or less the way I've tried to do it, but appreciate the clarification :)